packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -102,4 +110,116 @@ describe("deployments", () => {
+			  'Metadata.source': 'wrangler',
+			  'Metadata.created_on': '2021-01-01T00:00:00.000000Z',
+			  'Metadata.modified_on': '2021-01-01T00:00:00.000000Z',
+			  'resources.script': {
+			    etag: 'mock-e-tag',
+			    handlers: [ 'fetch' ],
+			    last_deployed_from: 'wrangler'
+			  },
+			  'resources.bindings': []
+			}
+
+						export default {
+							async fetch(request) {
+								return new Response('Hello World from Deployment 1701-E');
+							},
+						};"
+		`);
+			});
+		});
+
+		describe("rollback subcommand", () => {
+			it("should successfully rollback and output a message", async () => {
+				msw.use(
+					rest.put(
+						"*/account/:accountID/workers/scripts/:scriptName",
+						(req, res, ctx) => {
+							expect(req.url.searchParams.get("rollback_to")).toBe("Intrepid");
+
+							return res.once(
+								ctx.json(
+									createFetchResult({
+										created_on: "2222-11-18T16:40:48.50545Z",
+										modified_on: "2222-01-20T18:08:47.464024Z",
+										id: "space_craft_1",
+										tag: "alien_tech_001",
+										tags: ["hyperdrive", "laser_cannons", "shields"],
+										deployment_id: "galactic_mission_alpha",
+										logpush: true,
+										etag: "13a3240e8fb414561b0366813b0b8f42b3e6cfa0d9e70e99835dae83d0d8a794",
+										handlers: [
+											"interstellar_communication",
+											"hyperspace_navigation",
+										],
+										last_deployed_from: "spaceport_alpha",
+										usage_model: "intergalactic",
+										script: `addEventListener('interstellar_communication', event =\u003e
+							{ event.respondWith(transmit(event.request)) }
+							)`,
+										size: "1 light-year",
+									})
+								)
+							);
+						}
+					)
+				);
+
+				await runWrangler("deployments rollback Intrepid");
+				expect(std.out).toMatchInlineSnapshot(`
+			"🚧\`wrangler deployments\` is a beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose
+
+			Successfully rolled back to deployment ID: Intrepid
+			Rollbacks details:
+				{
+			  \\"created_on\\": \\"2222-11-18T16:40:48.50545Z\\",
+			  \\"modified_on\\": \\"2222-01-20T18:08:47.464024Z\\",
+			  \\"id\\": \\"space_craft_1\\",
+			  \\"tag\\": \\"alien_tech_001\\",
+			  \\"tags\\": [
+			    \\"hyperdrive\\",
+			    \\"laser_cannons\\",
+			    \\"shields\\"
+			  ],
+			  \\"deployment_id\\": \\"galactic_mission_alpha\\",
+			  \\"logpush\\": true,
+			  \\"etag\\": \\"13a3240e8fb414561b0366813b0b8f42b3e6cfa0d9e70e99835dae83d0d8a794\\",
+			  \\"handlers\\": [
+			    \\"interstellar_communication\\",
+			    \\"hyperspace_navigation\\"
+			  ],
+			  \\"last_deployed_from\\": \\"spaceport_alpha\\",
+			  \\"usage_model\\": \\"intergalactic\\",
+			  \\"script\\": \\"addEventListener('interstellar_communication', event =>/n/t/t/t/t/t/t/t{ event.respondWith(transmit(event.request)) }/n/t/t/t/t/t/t/t)\\",
+			  \\"size\\": \\"1 light-year\\"
+			}"
+		`);
```

Comment:
This is a placeholder until the message copy is determined for the success response. 
The prompt is also yet to be implemented.